service/service-samples/sample-service-rest-mysql-book/src/test/java/io/americanexpress/service/book/rest/model/SampleModelTest.java

```diff
@@ -0,0 +1,6 @@
+package io.americanexpress.service.book.rest.model;
+
+import io.americanexpress.synapse.framework.test.model.BaseModelsTest;
+
+public class SampleModelTest extends BaseModelsTest {
```

Comment:
does it need test cases? or do the tests just run from extending BaseModelsTest?

Reply:
It will run it from the parent class.